config/prod.toml

```diff
@@ -0,0 +1,69 @@
+[app]
+    appEnv                = "prod"
+    serviceName           = "metro"
+    shutdownTimeout       = 2
+    shutdownDelay         = 2
+    gitCommitHash         = "nil"
+[tracing]
+    serviceName           = "metro"
+    logSpans              = false
+    localAgentHostPort    = "localhost:6831"
```

Comment:
changed to host, port in latest code, should we leave this to default.toml value if not defined

Reply:
changed. Took latest master.